Create municipal-boundaries-final-update-for-2020 (#1617)
agrc/gis.utah.gov: _posts/2021-01-04-municipal-boundaries-final-update-for-2020.md

---

> title: 'Municipal Boundaries Final update for 2020'

Reviewer: ```suggestion
title: 'Municipal Boundaries Final Update for 2020'
```

author:
  display_name: mheagin
  email: mheagin
date: 2021-01-04 23:22:58
categories:
  - SGID Blog
tags:
  - municipal
  - boundaries
  - annexations
---

> Finally the end of 2020 is here. It has been a challenge for all of us. The hard working folks in the Lt. Governor's Office and all those working in the city and county Recorders Offices haven’t let that slow them down. Utah County once again leads the way with 30 annexations, and Cache County next at 13. Nine other counties follow with 4 to 6 annexations this year. We total out with 98 annexations this year. Not bad all things considered.

Reviewer: ```suggestion
Finally 2020 is behind us. Despite the year's challenges, the hardworking folks in the Lieutenant Governor's Office and all those working in the city and county recorders offices didn’t slow down. Utah County once again led the way with 30 annexations, and Cache County was next at 13. Nine other counties followed with 4 to 6 annexations throughout the year, bringing the grand total to 98 annexations in 2020, which is not bad, all things considered.
```

> The folks out in Erda in Tooele County voted to incorporate so we will be adding them in next year. Unfortunately we lost the Town of Cedar Heights, down in Iron County this year. 

Reviewer: ```suggestion
The folks out in Erda, in Tooele County, voted to incorporate, so we will be adding them in next year. Unfortunately we lost the Town of Cedar Heights, down in Iron County, this year. 
```

> As always feel free to check out all the changes and make sure your boundaries are where you think they should be, view them here: [Utah Municipality Modifications](https://www.arcgis.com/home/webmap/viewer.html?webmap=c5ab7e0fcd514f1a9db6b8dad55bba63).

Reviewer: ```suggestion
As always, feel free to check out all the municipal boundaries changes and make sure your boundaries are where you think they should be. You can view them on the [Utah Municipality Modifications page](https://www.arcgis.com/home/webmap/viewer.html?webmap=c5ab7e0fcd514f1a9db6b8dad55bba63).
```

> To review any specific annexation document, you can go here: [Utah Lt. Governor Municipal Certifications](https://municert.utah.gov/). 

Reviewer: ```suggestion
To review any specific annexation document, go to the [Utah Lt. Governor Municipal Certifications page](https://municert.utah.gov/). 
```

> Following are the changes and annexations since October 1, 2020 listed by County and City:

Reviewer: ```suggestion
The following table shows the changes and annexations (listed by county and city) since October 1, 2020:
```
